src/common/templates/_dbv3.tpl

```diff
@@ -347,14 +347,56 @@ OPTIONAL:
                 {{- $hostParts := split ":" $host }}
                 {{- $updatedHosts = append $updatedHosts $hostParts._0 }}
             {{- end }}
-            {{- $host = $updatedHosts }}
+            {{- $hosts = $updatedHosts }}
         {{- end }}
         {{- include "harnesscommon.dbconnection.connection" (dict "type" $dbType "hosts" $hosts "protocol" $protocol "extraArgs" $extraArgs "userVariableName" .userVariableName "passwordVariableName" .passwordVariableName "connectionType" "list") }}
     {{- else }}
         {{- fail (printf "ERROR: invalid contexts") }}
     {{- end }}
 {{- end }}
 
+{{/*
+Outputs Redis Port based on global and local values.
+
+USAGE:
+{{ include "harnesscommon.dbv3.redisConnection" (dict "ctx" $ "database" "cache") | quote }}
```

Comment:
Can you change the usage here, its misleading.

Reply:
Fixed. Copy error